core/src/main/java/org/infinispan/protostream/annotations/Proto.java

```diff
@@ -0,0 +1,20 @@
+package org.infinispan.protostream.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Defines a Protocol Buffers message without having to annotate all fields with {@link ProtoField}.
+ * Use this annotation to quickly generate messages from records or classes with public fields.
+ * Fields must be public and they will be assigned incremental numbers based on the declaration order.
```

Comment:
We should detail here that it's possible to overwrite fields with subsequent @ProtoField definitions.

Reply:
Done